### PR TITLE
Tt 32 jo itinerary creation

### PR DESCRIPTION
**Pull request by jordyob03**
Ticket: TT-32 

Changes
1. Front end
Added api to send data to backend

2. Itinerary service
Created main file, handlers, api endpoint and brought over db functions.

Notes
The itin creation UI and the itin table's fields don't completely match therefore some database fields will be left empty.

### CODE REVIEW

**Review by AbdulTur — APPROVED** [review]
LGTM Well done

File: frontend/src/pages/UserLogin.js
Comment (allymreid) [inline frontend/src/pages/UserLogin.js:1]: Is handleItinCllick used? Same logic is applied in navbar.js i think

**Review by allymreid — COMMENTED** [review]
We have duplicate itierary creation endpoints, (/itin-creation and /create-itinerary) and I made one small comment abput handling clickling the create itinerary button. Otherwise, LGTM! Thank you!

**Review by WyrdWyn4 — APPROVED** [review]
Looks good, we'll reconcile with Database in the other PR.
Great Work : )

**Pull request merged**